components/concept/toolbar/src/main/java/mozilla/components/concept/toolbar/Toolbar.kt

```diff
@@ -162,6 +162,11 @@ interface Toolbar {
      */
     fun addEditActionEnd(action: Action)
 
+    /**
+     * Hide end actions in edit mode.
```

Comment:
```suggestion
     * Toggles the visibility of the end actions in edit mode.
```
This isn't just hiding the, but toggling the visibility based on the provided [isVisible] value.
Can you also add an @param for `isVisible`?

Reply:
Removed this function.  Thanks